src/server/scripts/Events/pilgrims_bounty.cpp

```diff
@@ -532,16 +540,16 @@ class spell_pilgrims_bounty_turkey_tracker : public SpellScript
                 switch (stackAmount)
                 {
                     case 10:
-                        target->TextEmote("Turkey Hunter!", target, true);
+                        target->Talk(TURKEY_HUNTER, CHAT_MSG_RAID_BOSS_WHISPER, sWorld->getFloatConfig(CONFIG_LISTEN_RANGE_TEXTEMOTE), target);
```

Comment:
as they exist in the db all you need is 
```cpp
target->Talk(TURKEY_HUNTER, target);

Reply:
This is what the players themselves and themselves say.  Which data table should be used for the player's text？

Reply:
`target->Talk(TURKEY_HUNTER, target);`
Not available, others are missing ChatMsg parameters CHAT_MSG_RAID_BOSS_WHISPER

Reply:
CHAT_MSG_RAID_BOSS_WHISPER can't be right for player emote. surely it is CHAT_MSG_EMOTE

To which you can use

```cpp
target->TextEmote(EMOTE_TURKEY_HUNTER, target);
```

Reply:
nevermind I just saw the sniff `RaidBossWhisper`

Reply:
Blizzard is.. weird.

Reply:
but then we should fix it here
```
void Player::TextEmote(uint32 textId, WorldObject const* target /*= nullptr*/, bool /*isBossEmote = false*/)
```

I will open a PR and then we can adjust this accordingly. So we don't have to hardcode it...

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/18128 after merge this you update to 

```cpp
target->TextEmote(EMOTE_TURKEY_HUNTER, target, true);
```